dashboard/src/components/AppView/AppView.tsx

```diff
@@ -147,23 +158,32 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
   public handleEvent(e: MessageEvent) {
     const msg = JSON.parse(e.data);
     const resource: IResource = msg.object;
-    const key = `${resource.kind}/${resource.metadata.name}`;
+    const item = {
+      [resource.metadata.name]: { item: resource, isFetching: false },
```

Comment:
So the key is just the resource name and not the `resourceType/resourceName` ? In that case we will not be able to flatten resources in the future no?